pytest_github_actions_annotate_failures/plugin.py

```diff
@@ -11,7 +11,8 @@ def pytest_runtest_logreport(report):
         return
 
     # collect information to be annotated
-    filesystempath, lineno, _ = report.location
+    filesystem_relativepath, lineno, _ = report.location
+    filesystempath = filesystem_relativepath.replace('../', '')
```

Comment:
Do you think it might be a good idea to get the repo base path with `git`, then compute it relative to that?

Reply:
At first I thought I could resolve the path problem, but there might be a counterexample...